### PR TITLE
[WIP] Add thin arbiter  support

### PR DESCRIPTION
**Pull request by Madhu-1**
Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

**Describe what this PR does**
Provide some context for the reviewer

**Is there anything that requires special attention?**
Do you have any questions? Did you do something clever?

**Related issues:**
Fixes: #151

### CODE REVIEW

**Comment by humblec** [comment]
@madhu-1 thin arbiter support is in wip via https://github.com/gluster/gluster-csi-driver/pull/102 .

File: pkg/glusterfs/controllerserver.go [hunk 1]
```diff
@@ -99,6 +111,42 @@ func (cs *ControllerServer) ParseCreateVolRequest(req *csi.CreateVolumeRequest)
 	return &reqConf, nil
 }
 
+func validateThinArbiter(req *csi.CreateVolumeRequest) error {
+	if _, ok := req.Parameters["arbiterPath"]; ok {
```
Comment (Madhu-1) [inline pkg/glusterfs/controllerserver.go]: @JohnStrunk I have added `arbiterPath` to storage class to get the arbiter path  which will be like `<host:port/path/to/brick>` i thought  this cant be handled  with arbiterzones  as  you  suggested [here](https://github.com/gluster/gluster-csi-driver/pull/30/files#diff-c6b9b1189a45890edc0dcc6ed45f0205R102)

any suggestions from your side on this?

**Comment by Madhu-1** [comment]
closing this as @humblec working on this

**Comment by amarts** [comment]
@humblec 

> @Madhu-1 thin arbiter support is in wip via #102 .

I was thinking only 'thin-arbiter' is a good candidate for v1.0 release, and hence I guess this patch still makes sense. While you can work on #102 and get this part of the code overwritten if it makes sense then. With this approach, we will not have more options for users, and they will have a feature which is attractive enough to try out new GCS.

Let me know what you think?

**Pull request closed without merging**